### DISCUSSION TITLE
Light/Dark mode and Follow System Style toggles

### DISCUSSION THREAD

**Discussion by MobileAZN** [ryonakano/atlas#16] [Ideas]
Having the light/dark mode selector exposed, but hiding the "Follow system style" toggle hidden isn't super intuitive.  Maybe find a different way to present them, or have them together?  Or maybe hide the light/dark mode toggle if "Follow System Style" is enabled?

**Comment by ryonakano** [comment]
Hmm you're right, many softwares shows the style options parallel like:

- Light
- Dark
- Automatically (depending on the sun, sync with system, etc.)

Maybe we need a set of radio buttons like macOS?

![macos-big-sur-system-prefs-general-dark](https://user-images.githubusercontent.com/26003928/145334464-f53cdb04-0c67-4a92-92e8-4b0e67e22461.jpg)

(Image quoted & edited from https://support.apple.com/en-us/HT208976)

**Reply by ryonakano** [reply]
Locking conversation because now I created an Issue from this discussion. Let's discuss this here: #17
